### DISCUSSION TITLE
How to use nas drives to store photos?

### DISCUSSION THREAD

**Discussion by Cliff365** [immich-app/immich#10249] [General]
I'm not a Linux guy, I'm running Immich in Docker on my Win11 Pro box. It runs fine with the stock docker-compose.yml and .env file. For the life I cannot figure out how to upload and store photos to \\198.162.0.5\Synology_1\Immich\Uploads and \\198.162.0.5\Synology_1\Immich\Photos respectively.

I have read through guides, but I'm an idiot and I don't know what I'm reading most of the time. Can anyone help me out?

**Comment by githubedu** [comment]
My configuration is another, but the concept is that I mount the NAS/Server in a folder in the VM that is running Immich and point .env/docker to this folder.
I'm not sure with the terms in Windows, but If you try to map the NAS to local folder and on the Immich .env you put the folder location.  
In the .env you have the setting bellow:
```
# The location where your uploaded files are stored
UPLOAD_LOCATION=/photos
# The location where your database files are stored
DB_DATA_LOCATION=/photos/postgres
```

**Reply by Cliff365** [reply]
Thanks for posting that. My dilemma is how to show Immich the folders for uploads and photos reside on another device (192.168.0.5) on my lan?

**Reply by mmomjian** [reply]
You need to mount the network share on the system, this is not an Immich specific thing. Once it's mounted, you can set UPLOAD_LOCATION in the Immich env file to point to the network share

**Reply by Cliff365** [reply]
That's what I'm having problems with. I don't know the correct statements to put in my docker-compose.yml file to mount the network location.

**Reply by mmomjian** [reply]
It's not through docker compose. You can google "(operating system) mount network share guide"

**Reply by mmomjian** [reply]
It actually is possible through Docker but might be more challenging, depending on the network share type (NFS, Samba, etc): https://docs.docker.com/compose/compose-file/07-volumes/
